### ISSUE TITLE
Extra border on BarChartData

### ISSUE THREAD

**Issue by chengweiou** [imaNNeoFighT/fl_chart#143]
cannot set up custom border

```
BarChart(
            BarChartData(
              alignment: BarChartAlignment.spaceAround,
              barTouchData: BarTouchData(enabled: false),
              titlesData: FlTitlesData(
                show: true,
                leftTitles: SideTitles(
                  showTitles: false,
                ),
                rightTitles: SideTitles(
                  showTitles: true,
                  interval: 500,
                  getTitles: (v) => v.toInt().toString(),
                  textStyle: TextStyle(fontSize: 8, color: Colors.grey),
                ),
              ),
              barGroups: [
                BarChartGroupData(
                  x: 0,
                  barRods: [BarChartRodData(y: 2015, width: 25, isRound: false, color: Colors.lightBlueAccent)],
                  barsSpace: 10,
                ),
                BarChartGroupData(
                  x: 1,
                  barRods: [BarChartRodData(y: 2390, width: 25, isRound: false, color: Colors.lightBlueAccent)],
                ),
                BarChartGroupData(
                  x: 2,
                  barRods: [BarChartRodData(y: 2015, width: 25, isRound: false, color: Colors.lightBlueAccent)],
                ),
                BarChartGroupData(
                  x: 3,
                  barRods: [BarChartRodData(y: 2015, width: 25, isRound: false, color: Colors.lightBlueAccent)],
                ),
                BarChartGroupData(
                  x: 3,
                  barRods: [BarChartRodData(y: 1915, width: 25, isRound: false, color: Colors.lightBlueAccent)],
                ),
                BarChartGroupData(
                  x: 3,
                  barRods: [BarChartRodData(y: 1922, width: 25, isRound: false, color: Colors.lightBlueAccent)],
                ),
              ],
              borderData: FlBorderData(
                border: Border(top: BorderSide.none, right: BorderSide.none,)
              ),
              gridData: FlGridData(
                show: true,
                checkToShowHorizontalLine: (v) => v % 500 == 0,
              ),
            ),
          ),
```
focus on FlBorderData;
if show=false -> no border
if border: Border(top: BorderSide.none, right: BorderSide.none,) -> no different to default
if border: Border(top: BorderSide.none, right: BorderSide(color: Colors.grey),) -> will be two border on right side

**Comment by afinas-em** [comment]
try this.
 `borderData: FlBorderData(border: Border.all(width: 0.0))`

**Comment by imaNNeoFighT** [comment]
Sever bug from day one :)
Thanks for reporting, it has just been fixed, 
I will let you know when we released the new version, be patient.
Thanks!

**Comment by imaNNeoFighT** [comment]
Fixed in [0.6.3](https://github.com/imaNNeoFighT/fl_chart/releases/tag/0.6.3), check it out.
Thanks for reporting.

**Comment by chengweiou** [comment]
@imaNNeoFighT  fixed, thank you
